masterfabric-mobile/swift-camp
# CL - Text Module Navigation

## 📋 PR Description

This PR adds a Text module with navigation support from HomeView to TextView. The module follows the VIPER architecture for clean and structured implementation.

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Navigate to HomeView and click on the “Go to Text” button.
2. Verify that the TextView screen appears with the expected functionality and layout.

---

## 🔗 Related Links

- Documentation: https://medium.com/@dickareynaldisiahaan/viper-architecture-in-swift-building-robust-ios-apps-ed30d673d459

---

### 📷 Screenshots (Optional)

![Simulator Screenshot - iPhone 16 Pro - 2024-12-29 at 00 26 31](https://github.com/user-attachments/assets/236e4bb1-75a7-4c83-8df3-b0b7d0110934)

![Simulator Screenshot - iPhone 16 Pro - 2024-12-29 at 00 26 26](https://github.com/user-attachments/assets/6c0b5dab-6fd8-49de-bbff-fd0bb410adbc)
